libs/test/SalesforceSDKTest/src/com/salesforce/androidsdk/rest/RestClientTest.java

```diff
@@ -212,6 +215,15 @@ public void testGetResources() throws Exception {
         checkKeys(response.asJSONObject(), "sobjects", "search", "recent");
     }
 
+    /**
+     * Testing a get resources async call to the server - check response
+     * @throws Exception
+     */
+    public void testGetResourcesAsync() throws Exception {
```

Comment:
Adding a couple of tests that use client.sendAsync since client.sendAsync and client.sendSync are quite different. Only the former uses Volley queues.